api/BaseAudioContext.json

```diff
@@ -417,7 +417,7 @@
           "spec_url": "https://webaudio.github.io/web-audio-api/#dom-baseaudiocontext-createdelay",
           "support": {
             "chrome": {
-              "version_added": "14"
+              "version_added": "24"
```

Comment:
Should `DelayNode` also be updated?

Reply:
Definitely.  Unfortunately the tests for the audio nodes were not functioning because they didn't catch for the absence of these methods.  I've patched it now, and will follow up on these with the next collector update rounds!

Reply:
Looks like that was https://github.com/GooborgStudios/mdn-bcd-collector/commit/c91f8d17843ed5727734ea9905168a6325039bb6. That's been released so I suppose this PR could be updated now, but I understand if that's not trivial to do, so next round sounds good!